com.unity.cinemachine/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased] 
 - Bugfix: AxisState was not respecting timescale == 0
+- Bugfix: Very occasional axis drift in SimpleFollow when viewing angle is +-90 degrees.
```

Comment:
The bugfix message is not correct based on the jira ticket description. You said that small movements are the culprit not the viewing angle.

Reply:
It is correct.  It has to do with the viewing angle crossing the 90 degree boundary.

Reply:
From jira ticket:
"I was able to trigger this to happen with small mouse movements from any angle not just -90/90. So it seems like the angle is irrelevant other than the fact that if the bug is triggered the drift happens when viewing from the side and it won’t drift on its own when its near 0/180. Seems like just doing very small mouse movement rotations was enough to trigger it."

But I am ok with the changelog message, if you say it is correct. :)

Reply:
yeah, he meant "player angle in worldspace" which he at first thought was relevant, but it isn't